<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainSVMDlg</class>
 <widget class="QDialog" name="TrainSVMDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>189</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>189</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>189</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Train SVM Classifier</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Input Samples</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sampleFile">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="sampleBtn">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Output SVM File</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="svmFile">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="svmBtn">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gamma">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Penalty</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dC">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Training Ratio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="train_rate">
       <property name="minimumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>62</width>
         <height>22</height>
        </size>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.200000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>0.800000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="trainBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
